src/app/components/index.ts

```diff
@@ -16,17 +17,19 @@ export const components: any[] = [
   AccessibilityComponent,
   ServiceDownComponent,
   GetHelpComponent,
+  TermsAndConditionsRegisterOtherOrgComponent,
   TermsAndConditionsComponent,
   SignedOutComponent
 ];
 
-export * from './hmcts-global-header/hmcts-global-header.component';
-export * from './hmcts-global-footer/hmcts-global-footer.component';
 export * from '../../../projects/gov-ui/src/lib/components/hmcts-primary-navigation/hmcts-primary-navigation.component';
```

Comment:
not your code but src mentioned here

Reply:
The `projects > gov-ui` is a separate project that lives at the same level as `src`.

Reply:
Oh ok, necessary then I suppose